src/components/DelegationDashboard/DelegateCard.tsx

```diff
@@ -115,136 +158,273 @@ const DelegateCard = ({ delegate, className, trackNum, disabled }: Props) => {
+							<Image
+								src={'/assets/profile/w3f.svg'}
+								height={22}
+								width={22}
+								alt=''
+								className={'hidden sm:block'}
+							/>
+							<span className={`${poppins.variable} ${poppins.className} text-[10px] font-normal text-blue-dark-high sm:text-xs`}>W3F Delegate</span>
 						</div>
 					)}
 					{!!delegate?.dataSource?.includes(EDelegateSource.NOVA) && (
 						<div className='ml-[-0.6px] mr-[-0.6px] mt-[-1px] flex h-8 items-center gap-1 rounded-t-md border-[1px] border-solid border-[#3C74E1] bg-[#e2eafb] px-5 dark:bg-[#141C2D]'>
-							<ImageIcon
-								src='/assets/delegation-tracks/nova-wallet.svg'
-								alt='nova wallet icon'
+							<Image
+								src={'/assets/delegation-tracks/nova-wallet.svg'}
+								height={20}
+								width={20}
+								alt=''
+								className={'sm:hidden'}
 							/>
-							<span className='text-xs font-normal text-bodyBlue dark:text-blue-dark-high'>Nova Wallet Delegate</span>
+							<Image
+								src={'/assets/delegation-tracks/nova-wallet.svg'}
+								height={22}
+								width={22}
+								alt=''
+								className={'hidden sm:block'}
+							/>
+							<span className={`${poppins.variable} ${poppins.className} text-[10px] font-normal text-bodyBlue dark:text-blue-dark-high sm:text-xs`}>Nova Wallet Delegate</span>
 						</div>
 					)}
 					{!!delegate?.dataSource?.includes(EDelegateSource.PARITY) && (
-						<div className='ml-[-0.6px] mr-[-0.6px] mt-[-1px] flex h-8 items-center space-x-[6px] rounded-t-md border-[1px] border-solid border-[#7A67DF] bg-[#E4E1F9] px-5 dark:bg-[#25203D]'>
-							<ParityTechIcon />
-							<span className='text-xs font-normal text-bodyBlue dark:text-blue-dark-high'>Polkadot Delegate</span>
+						<div className='ml-[-0.6px] mr-[-0.6px] mt-[-1px] flex h-8 items-center space-x-[6px] rounded-t-md border-[1px] border-solid border-[#7A67DF] bg-[#E4E1F9] px-3 dark:bg-[#25203D] sm:px-5'>
+							<Image
+								src={'/assets/icons/polkadot-logo.svg'}
+								height={18}
+								width={18}
+								alt=''
+								className={'sm:hidden'}
+							/>
+							<Image
+								src={'/assets/icons/polkadot-logo.svg'}
+								height={22}
+								width={22}
+								alt=''
+								className={'hidden sm:block'}
+							/>
+							<span className={`${poppins.variable} ${poppins.className} text-[10px] font-normal text-bodyBlue dark:text-blue-dark-high sm:text-xs`}>Polkadot Delegate</span>
 						</div>
 					)}
 					{!!delegate?.dataSource?.includes(EDelegateSource.POLKASSEMBLY) && (
 						<div className='ml-[-0.6px] mr-[-0.6px] mt-[-1px] flex h-8 items-center gap-1 rounded-t-md border-[1px] border-solid border-pink_primary bg-[#FCE5F2] px-5 dark:bg-[#33071E]'>
-							<PolkadotIcon />
-							<span className='text-xs font-normal text-bodyBlue dark:text-blue-dark-high'>Polkassembly Delegate</span>
+							<Image
+								src={'/assets/delegation-tracks/pa-logo-small-delegate.svg'}
+								height={20}
+								width={20}
+								alt=''
```

Comment:
just use isMobile keyword to check display instead of hidden classaname and two times initialization of <Image